tests/php/Model/Delete/Fixtures/FilteredRecord.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace TractorCow\Fluent\Tests\Model\Delete\Fixtures;
+
+use SilverStripe\ORM\DataObject;
+use TractorCow\Fluent\Extension\FluentFilteredExtension;
+
+/**
+ * @mixin FluentFilteredExtension
+ */
+class FilteredRecord extends DataObject
```

Comment:
```suggestion
class FilteredRecord extends DataObject implements TestOnly
```

BYO namespace - same for the other test stubs

Reply:
Oh wow I forgot about that. Thanks for the reminder.